common/src/main/java/com/wynntils/features/inventory/ContainerSearchFeature.java

```diff
@@ -71,6 +79,13 @@ public class ContainerSearchFeature extends Feature {
     private SearchWidget lastSearchWidget;
     private SearchableContainerType currentSearchableContainerType;
     private boolean autoSearching = false;
+    private SearchQuery lastSearchQuery;
+
+    private final WynnItemFilterManager wynnItemFilterManager;
+
+    public ContainerSearchFeature() {
+        this.wynnItemFilterManager = new WynnItemFilterManager();
+    }
```

Comment:
Features don't use constructors like this, you can just declare this in-line?

Reply:
Oh god, this is a manager... Yeah, this is extra bad. Do the service refactor magicus mentioned, add this to Services.java and the mod will register it.

Reply:
I'll do this